app/posts/apply-for-teacher-training/2022-11-17-asking-candidates-if-they-received-free-school-meals.md

```diff
@@ -0,0 +1,121 @@
+
+Previously, we asked for:
+
+* sex
+* disability
+* ethnicity
+
+We were asked to investigate whether we could also add some questions to assess social-economic background. This is a hard-to-define measure which encompasses things such as family income, social class, education and the occupation of someone’s parents.
+
+## What we did
+
+We researched different ways of assessing social-economic background, and settled on a single question about whether someone received free school meals.
+
+![Screenshot showing a page with the title "Free school meals" followed by the text "Training providers will not see your answer to this question. This question helps the Department for Education understand the background of trainee teachers. Free school meals have generally only been available to children from families that receieve benefits." followed by the question "Did you ever get free school meals in the UK?" and 5 options: Yes, No, "I did not go to school in the UK", "I do not know" and "Prefer not to say".](free-school-meals-question.png "New question asking whether a candidate received free school meals")
+
+This is a proxy measure of social-economic status, as children have been eligible for free school meals if their families have been in receipt of certain benefits.
+
+The question is only shown to candidates who have British or Irish nationality, and who were born on or after 1 September 1964. This is because candidates who do not have British or Irish nationality are more likely to have gone to school outside of the UK, and candidates born before September 1964 are likely to have finished school before the free school meal policy was introduced in around 1980. These aren’t perfect criteria, as someone could still have gone to school in the UK despite not having British or Irish nationality, but are a compromise to avoid asking lots of people a question that is not relevant.
+
+As a candidate could have British or Irish nationality but still have gone to school outside the UK, we added a "I did not go to school in the UK" answer, and included "in the UK" within the question text.
+
+We added an "I do not know" answer if candidates cannot remember whether they received free school meals or not, or were unaware whether the school meals were free or paid for.
+
+The question includes the word "ever" as someone’s eligibility for free school meals may well have changed during their school years, and in this case we wanted to be clear that they should still select "Yes".
+
+We added some content so explain that, unlike the other existing equality and diversity questions, we are not revealing the answer to this question to providers if an application is accepted. This is because the information is not required by providers to submit data to the Higher Education Statistics Agency (HESA).
```

Comment:
```suggestion
We added some content to explain that, unlike the other existing equality and diversity questions, we are not revealing the answer to this question to providers if an application is accepted. This is because the information is not required by providers to submit data to the Higher Education Statistics Agency (HESA).
```